routing_model/index.html

```diff
@@ -531,6 +531,55 @@ <h2>Process for Adult/Elder promotion and demotion including merge</h2>
      We have to be careful that the section follows up on relocation once merged, so we may want to avoid active relocation when merging.
   </p>
 
+  <button class="collapsible">CheckElderTimeout</button>
+  <div class="content">
+    <p>As for incrementing work units, we want to update the eldership status of all nodes in a section on a syncrhonised, regular basis.<br/>
+    For this reason, it makes sense to have a timer going through Parsec.<br/>
+    Note that this timer has to be relatively fast so that it remains highly unlikely that 1/3 of the elders in any section would go offline within one timer's duration.
```

Comment:
`Fast`, not really my thought was probably 10 minutes or something.
This time is highly related to how fast we are happy for SectionInfo to change (i.e impact message relay etc..)

Note that this timer has to be only as fast as needed so that it remains highly unlikely that 1/3 of the elders in any section would go offline within one timer's duration.

Reply:
Yeah, I was thinking 5-10 mins too. Still: relatively fast to keep the low likelyhood of loosing too many elders is what I'm saying, so I'm not sure what you would change in the wording...